cfgov/cfgov/settings/base.py

```diff
@@ -657,6 +657,15 @@
     # https://github.com/turbolinks/turbolinks
     'TURBOLINKS': {},
 
+    # Teacher's Digital Platform Customer Review Tool
+    'TDP_CRTOOL': {},
+
+    # Teacher's Digital Platform Customer Review Tool Prototypes Pages
+    'TDP_CRTOOL_PROTOTYPES': {},
+
+    # Teacher's Digital Platform Search Interface Tool
+    'TDP_SEARCH_INTERFACE': {},
+
```

Comment:
If these are entirely replacing the existing `TDP_RELEASE` flag, could you please delete that original flag above? (Lines 651-654.)

If you want any of them to be turned on permanently on Beta, you should copy `TDP_RELEASE`'s hardcoded `site` condition.

Reply:
Nice, not sure how I missed that!  I will remove the TDP_RELEASE, and turn on the Beta flag for them.